core/src/main/resources/jenkins/security/whitelisted-classes.txt

```diff
@@ -1,11 +1,25 @@
+com.google.common.collect.AbstractListMultimap
+com.google.common.collect.AbstractMultimap
 com.google.common.collect.AbstractSetMultimap
+# Artifactory - https://issues.jenkins-ci.org/browse/JENKINS-48991
```

Comment:
No need to include comments like this. If it is a library bundled with core, no explanation is required, since we do not expect to want to remove the entry later.